imports/ui/pages/professor/manage_course.jsx

```diff
@@ -298,6 +306,17 @@ class _ManageCourse extends Component {
                       {this.props.course.allowStudentQuestions ? 'Disallow student questions' : 'Allow student questions'}
                     </div>
                   </div>
+                  {
+                    this.props.course.allowStudentQuestions
+                    ? <div className='btn-group btn-group-justified details-button-group'>
+                        <div className='btn btn-default' onClick={this.toggleRequireApprovedQuestions}>
+                        {
+                          this.props.course.requireApprovedQuestions ? 'Allow Unapproved Questions' : 'Require Questions to be Approved'
```

Comment:
Mention that this refers to public questions here as well on the button (perhaps also add a tooltip)